### PR TITLE
Update lib injection publishing after 1.0.0

### PR DESCRIPTION
**Pull request by devinsba**
# What Does This Do

# Motivation

# Additional Notes

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ deploy_latest_tag_to_docker_registries:
   rules:
     - if: '$POPULATE_CACHE'
       when: never
-    - if: '$CI_COMMIT_TAG =~ /^v.*/'
+    - if: '$CI_COMMIT_TAG =~ /^v1\..*/'
```
Comment (bantonsson) [inline .gitlab-ci.yml:145]: This should be documented in the major release repository workflow on the internal wiki

File: .github/workflows/lib-injection.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
+      - 'release/*'
       - 'releases/*'
```
Comment (bantonsson) [inline .github/workflows/lib-injection.yaml:7]: We have no branches named `releases/*`

Comment (devinsba) [reply .github/workflows/lib-injection.yaml]: Thanks, I wasn't sure if it was a typo from my first attempt of not.

**Review by bantonsson — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (devinsba) [reply .gitlab-ci.yml:145]: Done

**Pull request merged**